py/surveyqa/summary.py

```diff
@@ -120,31 +137,46 @@ def get_summarytable(exposures):
     darks = np.array([counts[i] for i in np.arange(2,len(counts), 4)])
     calibs = np.array([counts[i] for i in np.arange(3,len(counts), 4)])
 
+    #finding medians
+    med_air = get_median('AIRMASS', exposures)
+    med_seeing = get_median('SEEING', exposures)
+    med_exptime = get_median('EXPTIME', exposures)
+    med_transp = get_median('TRANSP', exposures)
+    med_sky = get_median('SKY', exposures)
+
     source = ColumnDataSource(data=dict(
         nights = list(Counter(exposures['NIGHT']).keys()),
         totals = list(Counter(exposures['NIGHT']).values()),
```

Comment:
The above medians are guaranteed to be in sorted order by NIGHT due to the `np.unique(...)` call in `get_median`, which returns the sorted unique values.  However, `Counter.keys()` does *not* guarantee that they will be in sorted order, though it does guaraneteed that `.keys()` and `.values()` will be matched together.  But your medians may not be matched up with the right nights.  You could use something like this:
```python
#- Count number of exposures per night
night_counter = Counter(exposures['NIGHT'])
nights = np.array(list(night_counter.keys()).keys()))
totals = np.array(list(night_counter.values()))
#- Ensure sorting by NIGHT
ii = np.argsort(nights)
nights = nights[ii]
totals = totals[ii]
```
Also check how `brights`, `grays`, `darks`, `calibs` is calculated to make sure that all columns actually match up.

Reply:
i'm currently working on this- should have the new pull request tonight or tomorrow